uamqp/errors.py

```diff
@@ -267,7 +266,7 @@ class MessageAccepted(MessageResponse):
 
 class MessageRejected(MessageResponse):
 
-    def __init__(self, condition=None, description=None, encoding='UTF-8'):
+    def __init__(self, condition=None, description=None, info=None, encoding='UTF-8'):
```

Comment:
Currently this would be breaking, if anyone had specified the call like:
`MessageRejected("foo", "bar", "ascii")`

And it seems that it's technically part of the public API.... however I think we're the only ones who call this right?
Users I think would only ever call `Message.reject()`?

Reply:
(I tried to make this feature non-breaking.. but looks like I still miss this here😣)

AFAIK, iot-sdk is depending on our uamqp library, chances are that they might use this API?
There're options if our intention is just for non-breaking:
- move to **kwargs
- putting `info` after `encoding`

Reply:
I'm pretty sure this is fine actually - I don't think a user would ever call this - and in the case of EH/IoT - I don't believe they ever reject messages anyway as they're already settled.

Even if a user did construct this object directly - I don't think there's any public API that accepts it as input....

Reply:
if that's the case, then I'm good with keeping my changes.

As the only internal place to construct `errors.MessageRejected` in uamqp is in `message.py:Message.reject` which makes me feel less concerned about this change.

Reply:
Chiming in with my usual dumb question: If this is breaking the public API, even if we have confident it's safe, should we bump the minor version # instead of patch?

Reply:
`info` has been moved to the end making it non-breaking